stix_shifter_modules/azure_sentinel/configuration/operators.json

```diff
@@ -0,0 +1,17 @@
+{
```

Comment:
Did you mean to have this file here in the configuration folder? It contains different (and extra) mappings from the operators.json included in the json folder.

Reply:
ohh this is a mistake. will remove it